html/modules/custom/itchefer_event_enrollment/src/Form/ProductModalForm.php

```diff
@@ -111,15 +111,15 @@ public function buildForm(array $form, FormStateInterface $form_state, $options
     $form['#attributes']['name'] = 'product_modal_form';
 
     // Radio buttons without options.
-    $form['radios'] = [
-      '#type' => 'radios',
+    $form['checkboxes'] = [
```

Comment:
The key should not depend on the UI controls being used, but reflect what is being selected (i.e. `products` or `product_ids`).